src/collective/easyform/fields.py

```diff
@@ -26,15 +25,36 @@
 from zope.schema.interfaces import IField
 
 
+@implementer(IForm)
+class GenericFormWrapper(object):
+    def __init__(self, view):
+        self.__view__ = view
+
+    def __getattr__(self, item):
+        return getattr(self.__view__, item)
+
+
 @implementer(IValidator)
 @adapter(IEasyForm, Interface, IEasyFormForm, IField, Interface)
-class FieldExtenderValidator(z3c_validator.SimpleFieldValidator):
-
+class FieldExtenderValidator(object):
     """ z3c.form validator class for easyform fields in the default fieldset"""
 
+    def __init__(self, context, request, view, field, widget):
+        self.context = context
+        self.request = request
+        self.view = view
+        self.field = field
+        self.widget = widget
+
     def validate(self, value):
         """ Validate field by TValidator """
-        super(FieldExtenderValidator, self).validate(value)
```

Comment:
I _think_ you still want the `super` call. The base class seems to do some [useful things](https://github.com/zopefoundation/z3c.form/blob/3.x/src/z3c/form/validator.py#L90-L108).
I fear that we could break other use cases when we do not use the base class.

Reply:
The StrictSimpleFieldValidator is by default registered for all fields so still gets called in the multi adapter call. This however allows someone else to prevent that happening if they want. Also it means this code isn't run twice. BTW. The tests fail without StrictSimpleFieldValidator validator being called.
I'll remove StrictSimpleFieldValidator as a subclass and put in a comment to make it more clear